proposals/0024/GOVERNANCE.md

```diff
@@ -102,8 +102,41 @@ The following apply to the repo or the part of the codebase for which one would
 - Assigned bugs related to area of expertise
 - May get a badge on PR and issue comments
 
+## Inactive members
+
+>This inactive members section follows the guideline defined in the Kubernetes community<sup>[5]</sup>
+
+_Members are continuously active contributors in the community._
+
+A core principle in maintaining a healthy community is encouraging active
+participation. It is inevitable that people's focuses will change over time and
+they are not expected to be actively contributing forever.
+
+However, being a member of the Tinkerbell GitHub organization comes with
+an [elevated set of permissions]. These capabilities should not be used by those
+that are not familiar with the current state of the Tinkerbell project.
+
+Therefore members with an extended period away from the project with no activity
+will be removed from the Tinkerbell Github Organization and will be required to
+go through the org membership process again after re-familiarizing themselves
+with the current state.
+
+### How inactivity is measured
+
+Inactive members are defined as members of the Tinkerbell Organization
+with **no** contributions across any repos within 8 months. This is
```

Comment:
We can stick with 8 months, but it won't be easy without tooling to easily tell this. The closest time ranges the [CNCF DevStats Activity page for Tinkerbell](https://tinkerbell.devstats.cncf.io/d/66/developer-activity-counts-by-companies?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All&var-companies=All) has are either the last year or last quarter.
<img width="508" alt="Screen Shot 2021-10-06 at 9 02 54 AM" src="https://user-images.githubusercontent.com/791000/136207668-e5b4b1fe-b3ac-472a-8819-7ddaba4fff7c.png">

Reply:
> [](https://github.com/tinkerbell/proposals/pull/50/files#diff-7f2439d5f7cdc18509ed1616be57259b19699dfd46b3b1675c469ee2289a8ce2)[](https://github.com/)measured by the CNCF [DevStats project].

Perhaps 1 year would be best. Again, this is not a blocker.